controller/cache/info.go

```diff
@@ -454,6 +456,13 @@ func populatePodInfo(un *unstructured.Unstructured, res *ResourceInfo) {
 		res.Info = append(res.Info, v1alpha1.InfoItem{Name: "Restart Count", Value: fmt.Sprintf("%d", restarts)})
 	}
 
+	// requests will be released for terminated pods either with success or failed state termination.
+
+	if reason != "Completed" && reason != "Error" {
+		res.Info = append(res.Info, v1alpha1.InfoItem{Name: podRequestsCPU, Value: fmt.Sprintf("%d", CPUReq.MilliValue())})
```

Comment:
You forgot to add the import

Reply:
missed it in a hurry, will refactor all these changes as soon as i get free time.